src/test/java/seedu/address/logic/commands/CommandResultTest.java

```diff
@@ -35,6 +35,44 @@ public void equals() {
         assertFalse(commandResult.equals(new CommandResult("feedback", false, true, false)));
     }
 
+    @Test
+    public void test_isShowHelp_returnsTrue() {
+        CommandResult commandResult = new CommandResult("Opened help window.", true, false, false);
+        assertTrue(commandResult.isShowHelp());
+    }
+
+    @Test
+    public void test_isShowHelp_returnFalse() {
+        CommandResult commandResult = new CommandResult("feedback", false, false, false);
+        assertFalse(commandResult.isShowHelp());
+    }
+
+    @Test
+    public void test_isExit_returnsTrue() {
+        CommandResult commandResult = new CommandResult("Exiting LibTask as requested ...", false,
+                true, false);
+        assertTrue(commandResult.isExit());
+    }
+
+    @Test
+    public void test_isExit_returnFalse() {
+        CommandResult commandResult = new CommandResult("feedback", false, false, false);
+        assertFalse(commandResult.isExit());
+    }
+
+    @Test
+    public void test_isPrevious_returnsTrue() {
+        CommandResult commandResult = new CommandResult("Here is your previous command.", false,
+                false, true);
+        assertTrue(commandResult.isPrevious());
+    }
+
+    @Test
+    public void test_isPrevious_returnFalse() {
+        CommandResult commandResult = new CommandResult("feedback", false, false, false);
+        assertFalse(commandResult.isPrevious());
+    }
```

Comment:
Great job on adding in additional test cases for the command result class. I like how it is done for details such as the help and exit commands. 👍